src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -142,7 +142,7 @@ function addGlobalModuleScriptTagsToDocument(callback: () => void) {
 
 function createEmscriptenModuleInstance(loadAssemblyUrls: string[], onReady: () => void, onError: (reason?: any) => void) {
   const module = {} as typeof Module;
-  const wasmBinaryFile = '_framework/wasm/mono.wasm';
+  const wasmBinaryFile = '_framework/wasm/dotnet.wasm';
```

Comment:
It might be useful to make this change as the dependency update PR. Changing this now would cause all of the E2E tests to fail. So let's get the https://github.com/dotnet/blazor/pull/1966 PR in first.